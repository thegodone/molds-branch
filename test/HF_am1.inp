%method
Method NDDO
Version AM1
end

*xyz 0 1
F        0.000000    0.000000    0.093782
H        0.000000    0.000000   -0.844036
*
