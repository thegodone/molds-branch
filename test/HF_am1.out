
                                 *****************
                                 * O   R   C   A *
                                 *****************

           --- An Ab Initio, DFT and Semiempirical electronic structure package ---

                  #######################################################
                  #                        -***-                        #
                  #  Department of molecular theory and spectroscopy    #
                  #              Directorship: Frank Neese              #
                  # Max Planck Institute for Chemical Energy Conversion #
                  #                  D-45470 Muelheim/Ruhr              #
                  #                       Germany                       #
                  #                                                     #
                  #                  All rights reserved                #
                  #                        -***-                        #
                  #######################################################


                         Program Version 3.0.3 - RELEASE   -


 With contributions from (in alphabetic order):
   Ute Becker             : Parallelization
   Dmytro Bykov           : SCF Hessian
   Dmitry Ganyushin       : Spin-Orbit,Spin-Spin,Magnetic field MRCI
   Andreas Hansen         : Spin unrestricted coupled pair/coupled cluster methods
   Dimitrios Liakos       : Extrapolation schemes; parallel MDCI
   Robert Izsak           : Overlap fitted RIJCOSX, COSX-SCS-MP3
   Christian Kollmar      : KDIIS, OOCD, Brueckner-CCSD(T), CCSD density
   Simone Kossmann        : Meta GGA functionals, TD-DFT gradient, OOMP2, MP2 Hessian
   Taras Petrenko         : DFT Hessian,TD-DFT gradient, ASA and ECA modules, normal mode analysis, Resonance Raman, ABS, FL, XAS/XES, NRVS
   Christoph Reimann      : Effective Core Potentials
   Michael Roemelt        : Restricted open shell CIS
   Christoph Riplinger    : Improved optimizer, TS searches, QM/MM, DLPNO-CCSD
   Barbara Sandhoefer     : DKH picture change effects
   Igor Schapiro          : Molecular dynamics
   Kantharuban Sivalingam : CASSCF convergence, NEVPT2
   Boris Wezisla          : Elementary symmetry handling
   Frank Wennmohs         : Technical directorship


 We gratefully acknowledge several colleagues who have allowed us to
 interface, adapt or use parts of their codes:
   Stefan Grimme, W. Hujo, H. Kruse, T. Risthaus : VdW corrections, initial TS optimization,
                                                   DFT functionals, gCP
   Ed Valeev                                     : LibInt (2-el integral package), F12 methods
   Garnet Chan, S. Sharma, R. Olivares           : DMRG
   Ulf Ekstrom                                   : XCFun DFT Library
   Mihaly Kallay                                 : mrcc  (arbitrary order and MRCC methods)
   Andreas Klamt, Michael Diedenhofen            : otool_cosmo (COSMO solvation model)
   Frank Weinhold                                : gennbo (NPA and NBO analysis)
   Christopher J. Cramer and Donald G. Truhlar   : smd solvation model


 Your calculation uses the libint2 library for the computation of 2-el integrals
 For citations please refer to: http://libint.valeyev.net

 This ORCA versions uses:
   CBLAS   interface :  Fast vector & matrix operations
   LAPACKE interface :  Fast linear algebra routines
   SCALAPACK package :  Parallel linear algebra routines


leaving
================================================================================
                                        WARNINGS
                       Please study these warnings very carefully!
================================================================================
Now building the actual basis set

WARNING: Direct SCF is incompatible with Method<>HF and Method<>DFT
  ===> : conventional SCF is chosen

WARNING: The NDO methods need Guess=HUECKEL or Guess=HCORE or Guess=MOREAD
  ===> : Guess is set to Hueckel

WARNING: The NDO methods need %rel::SOCType==1
  ===> : %rel::SOCType is set to 1

WARNING: The NDO methods cannot have frozencore=1
  ===> : %method FrozenCore=0 end


INFO   : the flag for use of LIBINT has been found!

================================================================================
                                       INPUT FILE
================================================================================
NAME = ../test/HF_am1.inp
|  1> %method
|  2> Method NDDO
|  3> Version AM1
|  4> end
|  5> 
|  6> *xyz 0 1
|  7> F        0.000000    0.000000    0.093782
|  8> H        0.000000    0.000000   -0.844036
|  9> *
| 10> 
| 11>                          ****END OF INPUT****
================================================================================

                       ****************************
                       * Single Point Calculation *
                       ****************************

---------------------------------
CARTESIAN COORDINATES (ANGSTROEM)
---------------------------------
  F      0.000000    0.000000    0.093782
  H      0.000000    0.000000   -0.844036

----------------------------
CARTESIAN COORDINATES (A.U.)
----------------------------
  NO LB      ZA    FRAG    MASS        X           Y           Z
   0 F     7.0000    0    18.998          0.000000000000000          0.000000000000000          0.177222296291403
   1 H     1.0000    0     1.008          0.000000000000000          0.000000000000000         -1.594996887170357

--------------------------------
INTERNAL COORDINATES (ANGSTROEM)
--------------------------------
 F      0   0   0   0.000000     0.000     0.000
 H      1   0   0   0.937818     0.000     0.000

---------------------------
INTERNAL COORDINATES (A.U.)
---------------------------
 F      0   0   0   0.000000     0.000     0.000
 H      1   0   0   1.772219     0.000     0.000

----------------------------
SLATER BASIS SET DIM=   5
----------------------------
  0 F     2 shells
 l=0 nsto= 1
    2       3.770820000000        1.000000000000
 l=1 nsto= 1
    2       2.494670000000        1.000000000000
  1 H     1 shells
 l=0 nsto= 1
    1       1.188078000000        1.000000000000

Checking for AutoStart:
The File: ../test/HF_am1.gbw exists
Trying to determine its content:
     ... Fine, the file contains calculation information
     ... Fine, the calculation information was read
     ... Fine, the file contains a basis set
     ... Fine, the basis set was read
     ... Fine, the file contains a geometry
     ... Fine, the geometry was read
     ... Fine, the file contains a set of orbitals
     ... Fine, the orbitals can be read
     => possible old guess file was deleted
     => GBW file was renamed to GES file
     => GES file is set as startup file
     => Guess is set to MORead
     ... now leaving AutoStart

------------------------------------------------------------------------------
                           ORCA NDO INTEGRAL CALCULATION
------------------------------------------------------------------------------

--------------
NDO PARAMETERS
--------------

Gamma integral treatment              ... MOPAC
Nuclear repulsuion treatment          ... AM1-style
Interaction factors:
s-s (sigma) =    1.0000
s-p (sigma) =    1.0000
s-d (sigma) =    1.0000
p-p (sigma) =    1.0000  p-p(pi) =    1.0000
p-d (sigma) =    1.0000  p-d(pi) =    1.0000
d-d (sigma) =    1.0000  d-d(pi) =    1.0000 d-d (delta) =    1.0000

--------------------------
Parameters for Element H :
--------------------------
 One-electron parameters (in eV)
  U(s)  =   -11.396427 Beta(s) =     6.173787 Neff(s) =     1.000000
 One-center electron repulsion parameters (in eV)
  G(s,s)=    12.848000
--------------------------
Parameters for Element F :
--------------------------
 One-electron parameters (in eV)
  U(s)  =  -136.105579 Beta(s) =    69.590277 Neff(s) =     2.000000
  U(p)  =  -104.889885 Beta(p) =    27.922360 Neff(p) =     5.000000 
 One-center electron repulsion parameters (in eV)
  G(s,s)=    16.920000
  G(s,p)=    17.250000 G(p,p)  =    15.510000
 Slater-Condon parameters (in eV)
 F2(p,p)=     7.5000 G1(s,p)=   14.4900

 Number of atoms                    ....    2
 Number of basis functions          ....    5

 Overlap integrals                  .... done
 One electron matrix                .... done
 Nuclear repulsion                  .... done
 Integral list                      .... done
 Electron-electron repulsion        .... done
-------------------------------------------------------------------------------
                                 ORCA SCF
-------------------------------------------------------------------------------

------------
SCF SETTINGS
------------
Hamiltonian:
 ZDO-Hamiltonian        Method          .... NDDO


General Settings:
 Integral files         IntName         .... ../test/HF_am1
 Hartree-Fock type      HFTyp           .... RHF
 Total Charge           Charge          ....    0
 Multiplicity           Mult            ....    1
 Number of Electrons    NEL             ....    8
 Basis Dimension        Dim             ....    5
 Nuclear Repulsion      ENuc            ....      3.0101449250 Eh

Convergence Acceleration:
 DIIS                   CNVDIIS         .... on
   Start iteration      DIISMaxIt       ....    12
   Startup error        DIISStart       ....  0.200000
   # of expansion vecs  DIISMaxEq       ....     5
   Bias factor          DIISBfac        ....   1.050
   Max. coefficient     DIISMaxC        ....  10.000
 Newton-Raphson         CNVNR           .... off
 SOSCF                  CNVSOSCF        .... on
   Start iteration      SOSCFMaxIt      ....   150
   Startup grad/error   SOSCFStart      ....  0.003300
 Level Shifting         CNVShift        .... on
   Level shift para.    LevelShift      ....    0.2500
   Turn off err/grad.   ShiftErr        ....    0.0010
 Zerner damping         CNVZerner       .... off
 Static damping         CNVDamp         .... on
   Fraction old density DampFac         ....    0.7000
   Max. Damping (<1)    DampMax         ....    0.9800
   Min. Damping (>=0)   DampMin         ....    0.0000
   Turn off err/grad.   DampErr         ....    0.1000
 Fernandez-Rico         CNVRico         .... off

SCF Procedure:
 Maximum # iterations   MaxIter         ....   125
 SCF integral mode      SCFMode         .... Conventional
 Integral Buffer length BufferLength    .... 1048576
 Integral index format  IndFormat       ....     0
 Integral value format  ValFormat       ....     0
 Integral Storage       Thresh          ....  1.000e-08 Eh

Convergence Tolerance:
 Convergence Check Mode ConvCheckMode   .... Total+1el-Energy
 Energy Change          TolE            ....  1.000e-06 Eh
 1-El. energy change                    ....  1.000e-03 Eh
 Orbital Gradient       TolG            ....  5.000e-05
 Orbital Rotation angle TolX            ....  5.000e-05
 DIIS Error             TolErr          ....  1.000e-06


Diagonalization of the overlap matrix:
Smallest eigenvalue                        ... 5.531e-01
Time for diagonalization                   ...    0.000 sec
Threshold for overlap eigenvalues          ... 1.000e-08
Number of eigenvalues below threshold      ... 0
Time for construction of square roots      ...    0.000 sec
Total time needed                          ...    0.001 sec

---------------------
INITIAL GUESS: MOREAD
---------------------
Guess MOs are being read from file: ../test/HF_am1.ges
Input Geometry matches current geometry (good)
Input basis set matches current basis set (good)
                      ------------------
                      INITIAL GUESS DONE (   0.0 sec)
                      ------------------

 InCore treatment chosen:
   Memory dedicated               ...    1024 MB
   Memory needed                  ...      0 MB
   Number of tiny    integrals    ...       0
   Number of small   integrals    ...       0
   Number of regular integrals    ...      22

--------------
SCF ITERATIONS
--------------
ITER       Energy         Delta-E        Max-DP      RMS-DP      [F,P]     Damp
                      *** Initiating the SOSCF procedure ***
                      *** Re-Reading the Fockian *** 
                      *** Removing any level shift *** 
ITER      Energy       Delta-E        Grad      Rot      Max-DP    RMS-DP
  0    -18.36175470 -18.3617546976  0.000002  0.000002  0.000003  0.000001
                 **** Energy Check signals convergence ****
              ***Rediagonalizing the Fockian in SOSCF/NRSCF***

               *****************************************************
               *                     SUCCESS                       *
               *           SCF CONVERGED AFTER   1 CYCLES          *
               *****************************************************


----------------
TOTAL SCF ENERGY
----------------

Total Energy       :          -18.36175470 Eh            -499.64875 eV

Components:
Nuclear Repulsion  :            3.01014493 Eh              81.91021 eV
Electronic Energy  :          -21.37189962 Eh            -581.55895 eV

One Electron Energy:          -35.92127924 Eh            -977.46770 eV
Two Electron Energy:           14.54937962 Eh             395.90875 eV


---------------
SCF CONVERGENCE
---------------

  Last Energy change         ...   -1.0001e-11  Tolerance :   1.0000e-06
  Last MAX-Density change    ...    2.0469e-06  Tolerance :   1.0000e-05
  Last RMS-Density change    ...    5.4155e-07  Tolerance :   1.0000e-06
  Last Orbital Gradient      ...    9.0199e-07  Tolerance :   5.0000e-05
  Last Orbital Rotation      ...    1.0953e-06  Tolerance :   5.0000e-05

             **** THE GBW FILE WAS UPDATED (../test/HF_am1.gbw) ****
             **** DENSITY FILE WAS UPDATED (../test/HF_am1.scfp.tmp) ****
             **** ENERGY FILE WAS UPDATED (../test/HF_am1.en.tmp) ****
----------------
ORBITAL ENERGIES
----------------

  NO   OCC          E(Eh)            E(eV) 
   0   2.0000      -1.758475       -47.8505 
   1   2.0000      -0.626485       -17.0475 
   2   2.0000      -0.513150       -13.9635 
   3   2.0000      -0.513150       -13.9635 
   4   0.0000       0.197007         5.3608 

                    ********************************
                    * MULLIKEN POPULATION ANALYSIS *
                    ********************************

-----------------------
MULLIKEN ATOMIC CHARGES
-----------------------
   0 F :   -0.315923
   1 H :    0.315923
Sum of atomic charges:    0.0000000

--------------------------------
MULLIKEN REDUCED ORBITAL CHARGES
--------------------------------
  0 F s       :     1.930049  s :     1.930049
      pz      :     1.385875  p :     5.385875
      px      :     2.000000
      py      :     2.000000
  1 H s       :     0.684077  s :     0.684077


                     *******************************
                     * LOEWDIN POPULATION ANALYSIS *
                     *******************************

----------------------
LOEWDIN ATOMIC CHARGES
----------------------
   0 F :   -0.278929
   1 H :    0.278929

-------------------------------
LOEWDIN REDUCED ORBITAL CHARGES
-------------------------------
  0 F s       :     1.904678  s :     1.904678
      pz      :     1.374250  p :     5.374250
      px      :     2.000000
      py      :     2.000000
  1 H s       :     0.721071  s :     0.721071


                      *****************************
                      * MAYER POPULATION ANALYSIS *
                      *****************************

  NA   - Mulliken gross atomic population
  ZA   - Total nuclear charge
  QA   - Mulliken gross atomic charge
  VA   - Mayer's total valence
  BVA  - Mayer's bonded valence
  FA   - Mayer's free valence

  ATOM       NA         ZA         QA         VA         BVA        FA
  0 F      7.3159     7.0000    -0.3159     0.9002     0.9002     0.0000
  1 H      0.6841     1.0000     0.3159     0.9002     0.9002     0.0000

  Mayer bond orders larger than 0.1
B(  0-F ,  1-H ) :   0.9002 

-------
TIMINGS
-------

Total SCF time: 0 days 0 hours 0 min 0 sec 

Total time                  ....       0.009 sec
Sum of individual times     ....       0.008 sec  ( 86.7%)

Fock matrix formation       ....       0.006 sec  ( 66.9%)
Diagonalization             ....       0.000 sec  (  0.8%)
Density matrix formation    ....       0.000 sec  (  0.1%)
Population analysis         ....       0.001 sec  (  8.9%)
Initial guess               ....       0.001 sec  (  6.9%)
Orbital Transformation      ....       0.000 sec  (  0.0%)
Orbital Orthonormalization  ....       0.000 sec  (  0.0%)
DIIS solution               ....       0.000 sec  (  0.1%)
SOSCF solution              ....       0.000 sec  (  3.0%)

-------------------------   --------------------
FINAL SINGLE POINT ENERGY       -18.361754697655
-------------------------   --------------------


                            ***************************************
                            *     ORCA property calculations      *
                            ***************************************

                                    ---------------------
                                    Active property flags
                                    ---------------------
   (+) Dipole Moment


------------------------------------------------------------------------------
                       ORCA ELECTRIC PROPERTIES CALCULATION
------------------------------------------------------------------------------

Dipole Moment Calculation                       ... on
Quadrupole Moment Calculation                   ... off
Polarizability Calculation                      ... off
GBWName                                         ... ../test/HF_am1.gbw
Electron density file                           ... ../test/HF_am1.scfp.tmp

-------------
DIPOLE MOMENT
-------------
                                X             Y             Z
Electronic contribution:     -0.00000      -0.00000       1.30225
Nuclear contribution   :      0.00000       0.00000      -1.05787
                        -----------------------------------------
Total Dipole Moment    :     -0.00000      -0.00000       0.24438
                        -----------------------------------------
Magnitude (a.u.)       :      0.24438
Magnitude (Debye)      :      0.62116


Timings for individual modules:

Sum of individual times         ...        0.072 sec (=   0.001 min)
STO integral calculation        ...        0.034 sec (=   0.001 min)  47.0 %
SCF iterations                  ...        0.038 sec (=   0.001 min)  53.0 %
                             ****ORCA TERMINATED NORMALLY****
TOTAL RUN TIME: 0 days 0 hours 0 minutes 0 seconds 326 msec
